index.js

```diff
@@ -14,6 +15,13 @@ const getPath = (prefix = '') => path.join(tempDir, prefix + uniqueString());
 
 const writeStream = async (filePath, data) => pipeline(data, fs.createWriteStream(filePath));
 
+const createTask = (tempyFunction, {extraArguments = 0} = {}) => async (...arguments_) => {
+	const [callback, options] = arguments_.slice(extraArguments);
+	const result = await tempyFunction(...arguments_.slice(0, extraArguments), options);
+	await callback(result);
+	await del(result, {force: true});
```

Comment:
https://github.com/sindresorhus/tempy/issues/11#issuecomment-674508737

Reply:
Since we're ignoring any failure, there is no longer any way for it to be activated since the created files and directories and deleted after the execution of the callback.